tools/firely-server-import.rst

```diff
@@ -0,0 +1,182 @@
+.. _tool_fsi:
+
+Firely Server Import (FSI)
+==========================
+
+**Firely Server Import (FSI)** is a CLI application designed to optimize massive resource ingestion into a Firely Server instance. In contrast to resource ingestion by sending HTTP requests to a running Firely Server instance, this tool writes data directly to the underlying FS database which increases the throughput significantly.
+
+.. note::
+    Currently, this tool only supports import into SQL Server database, however, we are working on support for MongoDB.
+
+.. Installation
+.. ------------
+.. TBD: describe the usage of dotnet install command
+
+General usage
+-------------
+
+.. attention::
+
+  * All Firely Server instances targeting the same database must be stopped while the import is performed.
+  * Only one instance of FSI per database can be run at a time.
+
+Prerequisites
+^^^^^^^^^^^^^
+The tool requires that the target SQL Server database already exists and contains all required tables and indexes. If you don't have a database with the schema yet, you first need to run the Firely Server as described in article :ref:`configure_sql`.
+
+
+Input files formats
+^^^^^^^^^^^^^^^^^^^
+
+FSI supports the following input file format:
+
+* FHIR *collection* bundles stored in ``*.json`` files, and
```

Comment:
transaction and batch bundles are also accepted, but treated as collection bundles.

Reply:
I suggest not to mention this because then we need to mention that the DELETE operation is not supported, etc. I think it's just better to omit this.